test/e2e/tests/account/incremental-security.spec.ts

```diff
@@ -0,0 +1,104 @@
+
+/**
+ * Sends ETH to the current account from the dapp send-eth-with-private-key-test
+ * This dapp is only used in this test spec, so we keep the function here
+ * TODO: migrate this function to follow page object modal pattern when this dapp is used in other tests
+ */
+async function sendEthWithPrivateKey(driver: Driver) {
+  const sendEthWithPrivateKeyButton = '#send';
+  await driver.waitForSelector(sendEthWithPrivateKeyButton);
+  await driver.pasteFromClipboardIntoField('#address');
+  await driver.clickElement(sendEthWithPrivateKeyButton);
+  await driver.waitForSelector(
+    { css: '#success', text: 'Success' },
+    { timeout: 15000 },
+  );
+}
+
+describe('Incremental Security', function (this: Suite) {
+  it('Back up Secret Recovery Phrase from backup reminder', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        title: this.test?.fullTitle(),
+        dappPath: 'send-eth-with-private-key-test',
+      },
+      async ({ driver, localNodes }: { driver: Driver; localNodes: any[] }) => {
+        // Seed Account
+        await localNodes[0].setAccountBalance(
+          '0x0Cc5261AB8cE458dc977078A3623E2BaDD27afD3',
+          '0x100000000000000000000',
+        );
+        await driver.navigate();
+
+        // agree to terms of use and start onboarding
+        const startOnboardingPage = new StartOnboardingPage(driver);
+        await startOnboardingPage.check_pageIsLoaded();
+        await startOnboardingPage.checkTermsCheckbox();
+        await startOnboardingPage.clickCreateWalletButton();
+
+        // skip collect metametrics
+        const onboardingMetricsPage = new OnboardingMetricsPage(driver);
+        await onboardingMetricsPage.clickNoThanksButton();
+
+        // create password
+        const onboardingPasswordPage = new OnboardingPasswordPage(driver);
+        await onboardingPasswordPage.check_pageIsLoaded();
+        await onboardingPasswordPage.createWalletPassword(WALLET_PASSWORD);
+
+        // secure wallet later
+        const secureWalletPage = new SecureWalletPage(driver);
+        await secureWalletPage.check_pageIsLoaded();
+        await secureWalletPage.skipSRPBackup();
+
+        // complete onboarding and pin extension
+        const onboardingCompletePage = new OnboardingCompletePage(driver);
+        await onboardingCompletePage.check_pageIsLoaded();
+        await onboardingCompletePage.completeOnboarding();
+
+        // copy the wallet address
+        const homePage = new HomePage(driver);
+        await homePage.check_pageIsLoaded();
+        await homePage.headerNavbar.clickAddressCopyButton();
+
+        // switched to Dapp and send eth to the current account
+        const windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        const testDapp = new TestDapp(driver);
+        await testDapp.openTestDappPage();
```

Comment:
instead of using the testDapp class, we'll then use the sendEthWithPrivKeyDapp class, so it results in a less confusing test

Reply:
agree to reduce confusio! addressed in commit [1748dc2](https://github.com/MetaMask/metamask-extension/pull/31564/commits/1748dc219a857e552ffb5507cc47318c5f1065a3)